.github/workflows/build.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       fail-fast: False
       matrix:
-        os: [ ubuntu-latest, macos-latest ]
+        os: [ ubuntu-latest, macos-latest, macos-14 ]
```

Comment:
macos-latest will currently default to macos-14 which is arm-64. To continue providing x86_64 and arm64 builds, I recommend this: `os: [ ubuntu-latest, macos-13, macos-14 ]`

Reply:
Good spot, fixed